plugins/async-node/jvm/src/test/kotlin/com/intuit/playerui/plugins/asyncnode/AsyncNodePluginTest.kt

```diff
@@ -0,0 +1,227 @@
+package com.intuit.playerui.plugins.asyncnode
+
+import com.intuit.hooks.BailResult
+import com.intuit.playerui.core.asset.Asset
+import com.intuit.playerui.utils.test.PlayerTest
+import com.intuit.playerui.utils.test.runBlockingTest
+import io.mockk.junit5.MockKExtension
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.suspendCancellableCoroutine
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.TestTemplate
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExtendWith(MockKExtension::class)
+internal class AsyncNodePluginTest : PlayerTest() {
+
+    val asyncNodeFlowSimple =
+        """{
+          "id": "counter-flow",
+          "views": [
+            {
+              "id": 'action',
+              "actions": [
+                {
+            "asset": {
+              "id": "action-0",
+              "type": "action",
+              "value": "{{foo.bar}}",
+            },
+          },
+          {
+            "id": "nodeId",
+            "async": true,
+          },
+              ],
```

Comment:
Properly format according to other formatting:
```suggestion
              "actions": [
                {
                  "asset": {
                    "id": "action-0",
                    "type": "action",
                    "value": "{{foo.bar}}",
                  },
                },
                {
                  "id": "nodeId",
                  "async": true,
                },
              ],
```

Reply:
done